src/rollover.py

```diff
@@ -95,7 +95,8 @@ def prompt_for_instances(ecs_instances, asg_contents, scale_down=False, sort_by=
         launch_time=dict(key=lambda i: i.launch_time, reverse=False),
     )
     sort_type = sorts[sort_by]
-    for x, instance in enumerate(sorted(ecs_instances, key=sort_type["key"], reverse=sort_type["reverse"])):
+    ecs_instances.sort(key=sort_type["key"], reverse=sort_type["reverse"])
```

Comment:
Ah, sorry. I realized that property. So you just want to make sure the list is still sorted whenever it's used thereafter?